portfolio/src/main/webapp/script.js

```diff
@@ -53,16 +53,27 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-// Creates a map and adds it to the page.
+// Creates a map and adds it to the page with markers on compost-dropoff locations.
 function initMap() {
-  // Coordinates center map  around Baltimore, MD
-  var myLatlng = new google.maps.LatLng(39.412, -76,775);
+  // Coordinates centers map  around NYC
+  var myLatlng = {lat: 40.763232, lng: -73.951047};
```

Comment:
The same as a previous comments (possibly in a different file), make these constants, with comments for why they're important.

Reply:
Done!